templates/internet-of-things/management.html

```diff
@@ -10,343 +10,493 @@
-                </p>
-              </div>
-              <div class="p-equal-height-row__item">
-                <hr class="p-rule--muted" />
-                <p>
-                  <a href="/landscape">Learn more about Landscape&nbsp;&rsaquo;</a>
-                </p>
-              </div>
-            </div>
-            <div class="p-equal-height-row__col">
-              <div class="p-equal-height-row__item">
-                <div class="p-section--shallow">
-                  {{ image(url="https://assets.ubuntu.com/v1/a64607f4-Snap_store_icon.png",
-                                    alt="Snap store",
-                                    width="69",
-                                    height="60",
-                                    hi_def=True,
-                                    loading="lazy") | safe
-                  }}
-                </div>
-              </div>
-              <div class="p-equal-height-row__item">
-                <h3 class="p-heading--5">Ubuntu Core &mdash; Dedicated Snapstore</h3>
-              </div>
-              <div class="p-equal-height-row__item">
-                <p>
-                  The Dedicated Snap Store is a private, and customisable platform for your applications. It's enterprise-ready and primed for efficient software distribution. It lets you utilise a huge range of our existing apps or build your own. Having a managed app store gives you role-based access controls, application versioning, OTA updates, controlled software rollouts, and much more.
-                </p>
-              </div>
-              <div class="p-equal-height-row__item">
-                <hr class="p-rule--muted" />
-                <p>
-                  <a href="/internet-of-things/appstore">Learn more about the Dedicated Snapstore&nbsp;&rsaquo;</a>
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <div class="p-section--shallow">
+            {{ image(url="https://assets.ubuntu.com/v1/dc17f4fd-microsoft-azure.png",
+                        alt="Microsoft Azure Logo",
+                        width="114",
+                        hi_def=True,
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="col-9 col-start-large-4 col-medium-4 col-start-medium-3">
+          <hr class="p-rule--muted" />
+
+          <div class="row">
+            <div class="col-6">
+              <div class="p-section--shallow">
+                <p class="p-heading--6">
+                  <i>
+                    “In collaboration with Canonical, we are making it simpler to reliably connect devices to Microsoft Azure IoT services. Snap support in Azure IoT Edge helps ensure consistent performance, enhanced security, and efficient updates across Linux distributions that support Snaps.”
```

Comment:
can't we use Vanilla pattern for this quote section?

Reply:
![image](https://github.com/user-attachments/assets/11cb827a-f446-44b6-ba58-f6ab6f826cfb)
I initially did that, but the underlying structure of the title header row doesn't expand the column for title even if the heading isn't link provided. So the title was wrapping in three lines which conflicted with the proposed design.